continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
```

Comment:
sorry I mean the default pipeline will include all tasks and in order of definition